cmd/cql/internal/wallet.go

```diff
@@ -59,36 +59,36 @@ func walletGen() {
 	if err != nil {
 		ConsoleLog.WithError(err).Error("unexpected error")
 		SetExitStatus(1)
-		return
+		return ""
 	}
 
-	fmt.Printf("wallet address: %s\n", keyHash.String())
+	return keyHash.String()
 
 	//TODO store in config.yaml
 }
 
 func runWallet(cmd *Command, args []string) {
 	configInit(cmd)
 
-	var err error
-	if tokenName == "" {
-		walletGen()
-		return
+	if conf.GConf.WalletAddress != "" {
+		fmt.Printf("\n\nwallet address: %s\n", conf.GConf.WalletAddress)
+	} else {
+		fmt.Printf("\n\nwallet address: %s\n", walletGen())
 	}
 
-	if strings.ToLower(tokenName) == "all" {
+	var err error
+	if strings.ToLower(tokenName) == "" {
 		var stableCoinBalance, covenantCoinBalance uint64
 
 		if stableCoinBalance, err = client.GetTokenBalance(types.Particle); err != nil {
```

Comment:
If the error is "no such balance", show the balance as 0, or tell the user "your account is not created in the TestNet, please apply tokens from our faucet first"